### PR TITLE
configurable run limits in `biscuit inspect`

### PR DESCRIPTION
**Pull request by divarvel**
```
biscuit inspect … --max-facts 100 --max-iterations 10 --max-time '10ms'
```

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -239,6 +239,28 @@ pub struct Inspect {
...
+        requires("verify-with-file")
+    )]
+    pub max_facts: Option<u64>,
+    #[clap(
+        long,
+        requires("verify-with"),
+        requires("verify-interactive"),
+        requires("verify-with-file")
+    )]
+    pub max_iterations: Option<u64>,
```
Comment (BastienVigneron) [inline src/cli.rs:255]: No comment for help content ?

File: src/cli.rs [hunk 2]
```diff
@@ -239,6 +239,28 @@ pub struct Inspect {
         conflicts_with("verify-interactive")
     )]
     pub verify_with: Option<String>,
+    #[clap(
+        long,
+        requires("verify-with"),
+        requires("verify-interactive"),
+        requires("verify-with-file")
+    )]
+    pub max_facts: Option<u64>,
```
Comment (BastienVigneron) [inline src/cli.rs:248]: No comment for help content ?

File: src/cli.rs [hunk 3]
```diff
@@ -239,6 +239,28 @@ pub struct Inspect {
...
+    )]
+    pub max_iterations: Option<u64>,
+    #[clap(
+        long,
+        requires("verify-with"),
+        requires("verify-interactive"),
+        requires("verify-with-file"),
+        parse(try_from_str = parse_duration)
+    )]
+    pub max_time: Option<Duration>,
```
Comment (BastienVigneron) [inline src/cli.rs:263]: No comment for help content ?

**Review by BastienVigneron — COMMENTED** [review]
It could be useful to add a little help text for each option (a simple comment with three slashes is enough with clap.rs).

File: src/cli.rs (hunk 1 above)
Comment (divarvel) [reply src/cli.rs:259]: right, i forgot. fixed!

File: src/cli.rs (hunk 2 above)
Comment (divarvel) [reply src/cli.rs:250]: done

**Review by BastienVigneron — APPROVED** [review]

**Pull request merged**